std/algorithm/searching.d

```diff
@@ -2852,6 +2864,76 @@ unittest
     assert(minPos!("a[0] < b[0]")(b) == [ [2, 4], [4], [4] ]);
 }
 
+import std.algorithm.comparison : min, max;
+
+/** Returns: Minimum Element in $(D range) or $(D unit) if $(D range) is empty.
+ */
+auto minElement(alias F = min, R)(R range,
```

Comment:
Do we really need these? "Compliance with C++ STL" doesn't seem like a good argument. I recommend moving these to a different PR where they can be evaluated separately.